README.md

```diff
@@ -72,4 +72,4 @@ You can now access the following URLs in your web browser:
 
 9. **DBdiagram**:
 Our Database Design
-![DB](https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/c2d8a65e-1019-4442-8d62-48efe1552033)
+![DB](https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/c2d8a65e-1019-4442-8d62-48efe1552033)
```

Comment:
is there any difference??

Reply:
Actually no have any different.

Reply:
Please care not to happen next time and I will remove it.